app-misc/radeontop/radeontop-1.3.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
You need to update the header (or use repoman commit)

Reply:
@mm1ke 
I did a change on the header by hand...
I am not sure if the change is ok.
Thank you for your review.